presto-main/src/test/java/com/facebook/presto/sql/planner/TestLocalExecutionPlanner.java

```diff
@@ -47,6 +47,13 @@ public void testCompilerFailure()
         assertFails("SELECT " + outer, COMPILER_ERROR);
     }
 
+    @Test
+    public void testCompiler()
```

Comment:
Neither name nor place does seem right.
I would just add this testcase to `AbstractTestQueries.testIn`, as it seems to fit there.